### ISSUE TITLE
automated email confirmation

### ISSUE THREAD

**Issue by lucarioams** [windingtree/stays-eth-barca#9]

**Comment by valera-rozuvan** [comment]
Can we not enable this for Barcelona? Can we make this work for Paris?

**Comment by lucarioams** [comment]
@valera-rozuvan yes unless it is very simple to enable based on the past work done on this

**Comment by valera-rozuvan** [comment]
Closing for now. Will be picked up as part of next version of Syays (using Videre protocol).
